test/extensions/filters/listener/tls_inspector/tls_inspector_integration_test.cc

```diff
@@ -166,7 +166,7 @@ TEST_P(TlsInspectorIntegrationTest, ContinueOnListenerTimeout) {
   // will continue wait. Then the listener filter timeout timer will be triggered.
   Buffer::OwnedImpl buffer("fake data");
   client_->write(buffer, false);
-  // the timeout is set as one seconds, sleep 5 to trigger the timeout.
+  // the timeout is set as one seconds, advance 2 seconds to trigger the timeout.
```

Comment:
Optional: capitalize first letter of the comment (even though it was already like that)

Reply:
done, thanks for speedy review